nuimo.py

```diff
@@ -307,7 +316,7 @@ def write_matrix(self, matrix, timeout, brightness=1.0):
         led_data = bytearray(matrix_bytes_list)
         led_uuid = self.characteristics_by_name['LED_MATRIX']['uuid']
         led_handle = self.value_handle(led_uuid)
-        self.write_by_handle(led_handle, bytes(led_data))
+        self.write_by_handle_async(led_handle, bytes(led_data), response)
```

Comment:
What's this extra `response` argument responsible for? You initialize it with a `Response()` object in line 22 but I can't yet see why we need it.